modules/administration/pages/tshoot-sync.adoc

```diff
@@ -32,3 +43,52 @@ Please run spacewalk-repo-sync in interactive mode to import it.
 
 To resolve the problem, you need to import the GPG key to {productname}.
 For more on importing GPG keys, see xref:administration:repo-metadata.adoc[].
+
+
+Checksum Mismatch::
+If a checksum has failed, you might see an error like this in the [path]``/var/log/rhn/reposync/*.log`` log file:
++
+----
+Repo Sync Errors: (50, u'checksums did not match
+326a904c2fbd7a0e20033c87fc84ebba6b24d937 vs
+afd8c60d7908b2b0e2d95ad0b333920aea9892eb', 'Invalid information uploaded
+to the server')
+The package microcode_ctl-1.17-102.57.62.1.x86_64 which is referenced by
+patch microcode_ctl-8413 was not found in the database. This patch has
+been skipped.
+----
++
+You can resolve this error by running the synchronization from the command prompt with the [command]``-Y`` option:
++
+----
+spacewalk-repo-sync --channel <channelname> -Y
+----
++
+This option verifies the repository data before the synchronization, rather than relying on locally cached checksums.
+
+
+Timeout::
+If you are synchronizing a {redhat} client with a very busy server, you might see a timeout error like this:
++
+----
+Errno 256 - No more mirrors to try
+----
++
+You can increase the timeout value in the [path]``/etc/rhn/spacewalk-repo-sync/yum.conf`` configuration file.
+Add or amend this line with a number of seconds:
++
+----
+[main]
+timeout=<value>
+----
++
+Another problem that can cause a timeout is an incomplete package download.
+To fix this, check the [path]``/var/spacewalk/packages/1/stage/`` directory and remove any partially downloaded file.
++
+If you use a cron job to run your synchronization automatically, use this code in your cron file to clean the stage directory before you begin:
++
+----
+# connect to customer center every day at random time
+# between 03:03 and 05:50
+3 3 * * * sleep $[ $RANDOM / 5 ]; rm /var/spacewalk/packages/1/stage/* && /usr/sbin/mgr-ncc-sync >/dev/null 2>/dev/null
```

Comment:
This command combination is a bad idea.

Every SUMA uses taskomatic to run sync on regular base. Having now cron doing the same is a bad idea as they can conflict. Beside of this, the mgr-ncc-sync is so outdated, i think it does not exist anymore.

Do not suggest a cron job and remove the mgr-ncc-sync command.
When there is a problem, just remove the packages from stage/1 and add a spacewalk-repo-sync example like in the other sections.

Reply:
Yeah, I had a feeling :wink:  Removed that bit.